soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -29,10 +31,56 @@ def __init__(self):
 
         self.add_state(SubmissiveDefender.State.marking,
                        behavior.Behavior.State.running)
+        self.add_state(SubmissiveDefender.State.clearing,
+                       behavior.Behavior.State.running)
 
         self.add_transition(behavior.Behavior.State.start,
                             SubmissiveDefender.State.marking, lambda: True,
                             "immediately")
+        self.add_transition(SubmissiveDefender.State.marking,
+                            SubmissiveDefender.State.clearing,
+                            lambda: self.go_clear,
+                            "when it is safe to clear the ball")
+        self.add_transition(
+            SubmissiveDefender.State.clearing,
+            SubmissiveDefender.State.marking,
+            lambda: self.subbehavior_with_name('kick-clear').state == behavior.Behavior.State.completed or not self.should_clear_ball(self.time_to_ball()),
+            "done clearing")
+
+    def time_to_ball(self):
+        max_vel = 3.5
+        max_accel = 1.8
+        delay = .1  #tune this better
+        rpos = self.robot.pos
+        bpos = main.ball().pos
+        #calculate time for self to reach ball using max_vel + a slight delay for capture
+        dist_to_ball = self.robot.pos.dist_to(main.ball().pos)
+        return (dist_to_ball / max_vel) + delay
+
+    def should_clear_ball(self, our_time_to_ball):
```

Comment:
Add a comment above the method giving a brief description of what it's doing - mainly that it returns true if our bot is closer than the closest opponent.